crates/rpc/rpc/src/debug.rs

```diff
@@ -492,7 +462,7 @@ where
         &self,
         opts: GethDebugTracingOptions,
         env: Env,
-        at: BlockId,
+        _at: BlockId,
```

Comment:
why is it unused now?

Reply:
ah this is no longer required because we don't need to spawn the js task with a provider at the targeted block

updated